wordpress-mobile/WordPress-Android
# [Platform] Fix screenshots lane: Update to x86_64 images

This PR fixes the `system-images` of emulators used by the `fastlane android screenshots` lane:

### Context

* The `screenshots` lane, which automates the capture of the app screenshot, starts by running `rebuild_screenshot_devices`, which restores the settings of emulators used for screenshots.
* Those emulator config files (located in `fastlane/emulators/*.ini` and copied over your `~/.android/avd/*.avd/config.ini` files during that lane) currently points to the `android-28/google_apis/x86` system-image

### Issue and Fix

The issue is, those `android-28;google_apis;x86` system-images are not available for download anymore. So the lane could work if you downloaded those system images a while ago and still have them on your machine, but won't work for people who never downloaded them, and won't work on CI either.

The fix modifies the emulator configs to **use x86_64 system-images instead** (`android-28;google_apis;x86_64`)

---

To test:

 * Start with a fresh install of AndroidSDK and the SDK – or alternatively, remove the `android-28/google_apis/x86` system-image from your installation if you had it installed previously a while ago back when it was still downloadable
 * Go into your SDK Manager and ensure that you have the system-image for {API 28, Google APIs, x86_64} downloaded
 * Run `bundle exec fastlane screenshots` and check that the lane succeeds (especially that you don't have a `PANIC: Broken AVD` error)

---

PR submission checklist:
